index.html

```diff
@@ -0,0 +1,737 @@
+                        </li>
+                    </ul>
+                </div>
+            </li>
+            <li class="u-col-xs-12 u-col-m-4 u-col-l-3">
+                <div class="story story--smallMobile">
+                    <div class="lazy story-image lazyLoaded"> </div>
+                    <h4 class="story-title">
+                        <a href="#">
+                            Here's what the iPhone 11 lineup costs in India - and what you can buy instead                        </a>
+                    </h4>
+                    <ul class="story-byline">
+                        <li><a href="#">Ivan Mehta</a></li>
+                        <li>
+                            <time datetime="2019-09-11T06:11:29+00:00" class="timeago" data-full-date="Sep 11, 2019 - 09:11">16 days ago</time>
+                        </li>
+                    </ul>
+                </div>
+            </li>
+            <li class="u-col-xs-12 u-col-m-4 u-col-l-3">
+                <div class="story story--smallMobile">
+                    <div class="lazy story-image lazyLoaded"> </div>
+                    <h4 class="story-title">
+                        <a href="#">
+                            Apple details its new trade-in program and Watch Studio                        </a>
+                    </h4>
+                    <ul class="story-byline">
+                        <li><a href="#">Rachel Kaser</a></li>
+                        <li>
+                            <time datetime="2019-09-11T00:24:29+00:00" class="timeago" data-full-date="Sep 11, 2019 - 03:24">17 days ago</time>
+                        </li>
+                    </ul>
+                </div>
+            </li>
+            <li class="u-col-xs-12 u-col-m-4 u-col-l-3">
+                <div class="story story--smallMobile">
+                    <div class="lazy story-image lazyLoaded"> sizes="(max-width: 513px) 100vw, 513px" width="513" height="270" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" style="background-image: url(&quot;https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/09/iPhone-11-pricing-513x270.jpg&quot;);"></div>
+                    <h4 class="story-title">
+                        <a href="#">
+                            Apple details its new trade-in program and Watch Studio                        </a>
+                    </h4>
+                    <ul class="story-byline">
+                        <li><a href="#">Rachel Kaser</a></li>
+                        <li>
+                            <time datetime="2019-09-11T00:24:29+00:00" class="timeago" data-full-date="Sep 11, 2019 - 03:24">17 days ago</time>
+                        </li>
+                    </ul>
+                </div>
+            </li>
+        </ul>
+    </section>
+
+    <section class="funding-article">
```

Comment:
#### :bulb:  A `section` element should have a heading element (`h2`-`h6`). 

Why are we using `nav` as a child of this section instead of a heading element? :thinking: 

Please, learn more about [`section` element here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section). Also note this section :point_down: 
  > ### Usage notes
  > - Each `<section>` should be identified, typically by including a heading (`<h1>`-`<h6>` element) as a child of the `<section>` element.
  > - If it makes sense to separately syndicate the content of a `<section>` element, use an `<article> `element instead.
  > - Do not use the `<section>` element as a generic container; this is what `<div>` is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document.

#### [Source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#Usage_notes)